### PR TITLE
Sensu Go API docs: Get and get all endpoints

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Adds API-wide information to the overview page
- Adds a page for each API endpoint
- Adds docs for get and get all endpoints, not including special types of get request. See https://github.com/sensu/sensu-docs/issues/580 for specific endpoints included in this PR

To do:

- [ ] Include info from https://github.com/sensu/sensu-go/issues/2252

## Motivation and Context

Partially addresses 580

### CODE REVIEW

File: content/sensu-go/5.0/api/assets.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+The `/assets` API endpoint provides HTTP GET access to [asset][1] data.
+
+#### EXAMPLE {#assets-get-example}
+
+The following example demonstrates a request to the `/assets` API, resulting in
+a JSON Array containing [asset definitions][1].
+
+{{< highlight shell >}}
+curl -s http://127.0.0.1:8080/apis/core/v2/namespaces/default/assets -H "Authorization: Bearer TOKEN"
```
Comment (palourde) [inline content/sensu-go/5.0/api/assets.md]: We are going with `/api/` instead of `/apis/`, sorry about that!

Comment (apaskulin) [reply content/sensu-go/5.0/api/assets.md]: Fixed!

File: content/sensu-go/5.0/api/overview.md [hunk 2]
```diff
@@ -8,6 +9,63 @@ menu:
...
+- `{namespace}` is the namespace name. The examples in these API docs use the `default` namespace.
+
+### Data format
+
+The API uses JSON formatted requests and responses.
+In terms of [sensuctl output types][1], the Sensu API uses the `json` format, not `wrapped-json`.
+
+### Data size
+
+API responses are limited to 0.512 MB in size.
```
Comment (palourde) [inline content/sensu-go/5.0/api/overview.md]: This is actually only for the request body size (i.e. the payload when posting to the API).

File: content/sensu-go/5.0/api/cluster.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+The `/cluster/members` API endpoint provides HTTP GET access to [Sensu cluster][1] data.
+
+#### EXAMPLE {#clustermembers-get-example}
+
+The following example demonstrates a request to the `/cluster/members` API, resulting in
+a JSON Array containing a Sensu cluster definition.
+
+{{< highlight shell >}}
+curl -s http://127.0.0.1:8080/api/core/v2//cluster/members -H "Authorization: Bearer TOKEN"
```
Comment (palourde) [inline content/sensu-go/5.0/api/cluster.md]: There's an extra `/` between v2 and cluster

**Review by palourde — CHANGES_REQUESTED** [review]
This is _really_ good. There's only 2 small details, otherwise :shipit:

For reference, it's also possible to list namespaced resources (i.e. `http://127.0.0.1:8080/api/core/v2/namespaces/default/assets`) across namespaces (equivalent of `--all-namespaces` flag) by removing the namespaces collection (i.e. `http://127.0.0.1:8080/api/core/v2/assets`), however it requires the user to have permissions granted for the resource via a cluster role binding and a cluster role. Not sure if it's worth mentioning, and if so, where.

**Review by palourde — APPROVED** [review]
👌

**Comment by calebhailey** [comment]
EXCITE! 🙌

**Pull request merged**